src/Microsoft.ML.Data/StaticPipe/StaticPipeUtils.cs

```diff
@@ -13,7 +14,9 @@ namespace Microsoft.ML.Data.StaticPipe.Runtime
 {
     /// <summary>
     /// Utility methods for components that want to expose themselves in the idioms of the statically-typed pipelines.
-    /// These utilities are meant to be called by and useful to component authors, not users of those components.
+    /// These utilities are meant to be called by and useful to component authors, not users of those components. The
+    /// purpose is not to keep them hidden per se, but rather out of the face of users that are just trying to use the
```

Comment:
>out of the face of users [](start = 62, length = 24)

out of the attention of users

Reply:
Yes I did not describe this well -- I think the flaw in my writing goes a bit beyond simple word choice. Let me rephrase this sentence entirely.

---
In reply to: [216475276](https://github.com/dotnet/machinelearning/pull/869#discussion_r216475276) [](ancestors = 216475276)